### PR TITLE
Wrap config properties in 'service' object

### PR DESCRIPTION
**Pull request by mrbrowning**
This is necessary for install from the UI to work.

### CODE REVIEW

**Comment by mgummelt** [comment]
> This is necessary for install from the UI to work.

Why do they expect "service"?

**Comment by mrbrowning** [comment]
@mgummelt they don't expect `service` per se, but the top-level configuration items have to be objects themselves (I guess so that they can be grouped into tabs) and not scalar values.

**Comment by mrbrowning** [comment]
@mesosphere-ci retest this please

**Comment by mrbrowning** [comment]
@mesosphere-ci retest this please

Cluster launch is failing.

**Comment by mgummelt** [comment]
known problem: https://jira.mesosphere.com/browse/SPARK-448

File: history/package/config.json [hunk 1]
```diff
@@ -1,62 +1,69 @@
...
-            "default": "{{default-docker-image}}"
-        },
-        "hdfs-config-url": {
-            "type": "string",
-            "description": "URL which serves hdfs-site.xml and core-site.xml (i.e. <hdfs-config-url>/hdfs-site.xml should exist)"
-        },
-        "cleaner": {
+        "service": {
             "type": "object",
+            "description": "spark-history configuration properties",
```
Comment (mgummelt) [inline history/package/config.json]: DC/OS Apache Spark History Server configuration properties.

**Comment by mgummelt** [comment]
retest this please

**Comment by mrbrowning** [comment]
@mgummelt verdict?

**Comment by mgummelt** [comment]
One comment regarding the copy I'd like you to address, then g2g

**Pull request merged**